tests/generator-Tests/Unit-Tests/ManagedTests.cs

```diff
@@ -250,5 +274,35 @@ public void StripArity ()
 			Assert.AreEqual ("System.Collections.Generic.Dictionary<T,System.Collections.Generic.List<System.String>>", @class.Methods [0].ReturnType);
 			Assert.AreEqual ("System.Collections.Generic.Dictionary<T,System.Collections.Generic.List<System.String>>", @class.Methods [0].ManagedReturn);
 		}
+
+#if NET6_0_OR_GREATER
+		// Our net472 tests aren't using C# 8.0
```

Comment:
Should we get our net472 tests onto C# 8?  We've done that for other assemblies which target netstandard or net472, e.g. PR #913.

Reply:
Yeah, that was easy enough.  Done!